Common/src/config_structure.cpp

```diff
@@ -243,6 +243,8 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   /*!\brief FLUID_MODEL \n DESCRIPTION: Fluid model \n OPTIONS: See \link FluidModel_Map \endlink \n Default: STANDARD_AIR \ingroup Config*/
   addEnumOption("FLUID_MODEL", Kind_FluidModel, FluidModel_Map, STANDARD_AIR);
 
+  /* DESCRIPTION: Automatically reorient elements that seem flipped */
```

Comment:
Small, nitpicky note: for keeping the config_structure.cpp file organized; this could be moved to the part of this routine marked " Options related to input/output files and formats" in the comments. 
Similar organization exists in the config_template.cfg file.